translations/ui-bulk-edit/en.json

```diff
@@ -437,7 +437,8 @@
   "actionMenu.placeholder": "Search column name",
 
   "previewModal.message": "{count, number} records will be changed if the Commit changes button is clicked. You may choose Download preview to review all changes prior to saving.",
-  "previewModal.message.empty.marc": "All instances have source FOLIO. Use “Instances and  Administrative data” option for bulk edit.",
+  "previewModal.message.empty.notSupported": "All instances have source that is not supported by MARC records bulk edit and cannot be updated.",
```

Comment:
This phrasing is a little awkward. Maybe sth like "No instances can be updated because none have source MARC." or "Bulk edit can only update instance records with a source of MARC but no records match."?